app/views/markets/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<!-- <% render partial: 'index' %> -->
+<h1> MARKETS </h1>
+<div class ="new_market_button">
+  <%= button_to "Create a new market", new_market_path, method: :get %>
+</div>
+
+  <% @markets = Market.all.order(:name) %>
```

Comment:
Again, this processing of data should happen in the controller, not here in the view.